iree/tools/CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ if(IREE_ENABLE_EMITC)
   )
 endif()
 
+configure_file(config.txt.in config.txt)
```

Comment:
Let's use more descriptive names for this file.

How about `build_config_template.txt.in` -> `build_config.txt`?

Reply:
Renamed `config` -> `build_config`.  But didn't add `_template`, because it is a common idiom that a `.in` suffix means a template.

Reply:
I had consistency with https://github.com/google/iree/blob/main/iree/hal/cts/cts_test_template.cc.in and https://github.com/google/iree/blob/main/iree/tools/android/run_module_app/AndroidManifest.xml.template in mind with "template" in the file name, but `.in` might be enough, yeah.

Reply:
oh okay, internal consistency does matter. changed.